modules/graphql_views/src/Plugin/Deriver/ViewDeriver.php

```diff
@@ -128,7 +128,7 @@ public function getDerivativeDefinitions($basePluginDefinition) {
         'view' => $viewId,
         'display' => $displayId,
         'paged' => $paged,
-        'cache_tags' => $view->getCacheTags(),
+        'cache_tags' => $view->getExecutable()->getCacheTags(),
```

Comment:
Note: I think the right approach would be to fetch the cache tags from $view->element['#cache']['tags'] after the execution. There might be dynamic cache tags, which are not taken into account here.